whisparr.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Whisparr
+description: Whisparr wiki homepage
+published: true
+date: 2022-03-14T03:22:15.199Z
+tags: whisparr
+editor: markdown
+dateCreated: 2021-05-16T13:53:17.360Z
+---
+
+![128.png](/assets/whisparr/logos/128.png)
+
+> Whisparr is a movie collection manager for Usenet and BitTorrent users. It can monitor multiple RSS feeds for new movies and will interface with clients and indexers to grab, sort, and rename them. It can also be configured to automatically upgrade the quality of existing files in the library when a better quality format becomes available.
```

Comment:
```suggestion
> Whisparr is an adult movie collection manager for Usenet and BitTorrent users. It can monitor multiple RSS feeds for new movies and will interface with clients and indexers to grab, sort, and rename them. It can also be configured to automatically upgrade the quality of existing files in the library when a better quality format becomes available.
```

Reply:
@bakerboy448 you were just ready weren't you